### PR TITLE
ISPN-5254 Server not always stopped properly with the IBM JDK

### PR DESCRIPTION
**Pull request by chepa653**

### CODE REVIEW

**Comment by chepa653** [comment]
Hey @danberindei! Can you please review it?

File: integrationtests/as-integration-client/build.xml [hunk 1]
```diff
@@ -51,44 +51,128 @@
...
-      </exec>
-      <exec executable="taskkill" osfamily="windows">
-         <arg line="/F /T /PID ${pid}"/>
-      </exec>
-      <delete file="jps.pid"/>
-   </target>
+    <target name="-do.kinit" depends="-check.skipped" unless="tests.skipped">
+        <exec executable="bash" output="ps.pid" resultproperty="ps.result" failifexecutionfails="false" failonerror="false" osfamily="unix">
+            <arg value="-c"/>
+            <arg value="ps -eaf --columns 20000 | grep jboss-modules.jar | grep -v -w grep | awk '{print $$2;}'"/>
```
Comment (danberindei) [inline integrationtests/as-integration-client/build.xml]: `ps -fCjava` should filter by command name and avoid the need for `grep -v -w grep` - unless `-C` is not supported on one of the OSes we test on.

**Comment by danberindei** [comment]
We also have a `kill-jbossas.xml` build script in `server/integration/testsuite` that does exactly the same thing. Ideally we should have only one, but otherwise we need to replicate the changes there as well.

**Comment by chepa653** [comment]
Hey @danberindei ! Added new changes. I cant use ps -Cf because Solaris does not have such option

**Comment by danberindei** [comment]
@chepa653 oh well, I guess we'll have to keep it like that then.

**Comment by galderz** [comment]
@chepa653 What's up with the duplication highlighted by @danberindei? Is that correct? Can we sort it out?

**Comment by danberindei** [comment]
Integrated, thanks Vitalii!

@galderz yeah, it would be nice to unify the Ant build scripts, but I found a couple more IBM java processes lying around on the build agent machine this morning, so I'm integrating this PR as is.

**Pull request closed without merging**